main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -238,7 +238,8 @@
 
     @Override
     public void onCreate(final Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState, R.layout.cachedetail_activity);
```

Comment:
There were contexts, where this didn't work in the past and for some Android versions. Did you cross-check on light theme and older versions?

Reply:
No. But the oncreate method deleted was not even from Android system (it was c: geo self-made, despite the name oncreate suggesting otherwise) and I created the new method setthemeandcontentview to execute the new parts that were added in custom oncreate method. 

Cross-check can be done

Reply:
Hi @bekuno I tested this version on Android 5 (SDK21, Emulator) using both light and dark skin and clicking through all of the important dialogs. It seems to work.

Is there any specific situation where this did not work in the past that I should test additionally? Otherwise I would merge this.

Reply:
Sorry, I meant @rsudev before, not @bekuno ...

Reply:
As it is quite some time ago I do not remember the details any more. As the new implementation changes the call order for initializing theme and contentview with respect to the progress setting and the up action, those were the parts I would look into specifically.
Regarding the progress indicator, I am not up to date whether it is still in use at all (as it was phased out AFAIR by Google quite some versions ago). Previously the main usage context of this was the map, but that has been changed, if I am not mistaken.

Reply:
Hi @rsudev , I checked progress bar in Android 5 SDK21 light mode, and there is indeed a problem: there is a black bar around it. However this bar is also present when using the master version, so I assume it is not introduced by the lint fixes

![image](https://user-images.githubusercontent.com/6909759/112435015-77ac3880-8d44-11eb-86b5-ad1b3e6d8d5a.png)

Is this what you meant?

Reply:
Now, that is the progress dialog (and those glitches are more likely have been introduced by recent theme changes rather than the change at hand).
In former Android version it was possible to have a progress indicator embedded in the ActionBar and that was used for the map update indication (and IIRC a few other places). But as I said, usage of that might be gone in the meantime.
If the up navigation (tapping the arrow in the upper-left corner) is still working as expected I would consider my concerns addressed.

Reply:
@rsudev ok the up navigation works. I will merge this PR